src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -60,9 +58,31 @@ def fake_quantize(
     bit_range = 2**args.num_bits
     max_q = torch.tensor(bit_range / 2 - 1, device=x.device)
     min_q = torch.tensor(-bit_range / 2, device=x.device)
-    Q = torch.zeros_like(x)
-    Q = quantize(x, scale, zero_point, min_q, max_q)
-    return dequantize(Q, scale, zero_point)
+
+    columns = x.shape[1]
```

Comment:
does this line up to input channels?

Reply:
Only for group_size when it is used. 
There is a to-do in the future for more generic case